### PR TITLE
fix css toggle-button ignores title translation

### LINKED ISSUE #43: Open/Close toggle button is hardcoded and cannot be translated
__Describe the Bug__

Unfortunately, the current implementation ignores translations and the buttons texts are hardcoded inside the css. 
The easiest fix is to use the title attribute of the parent div that is already translated.

I made a pr: #44

### PR DESCRIPTION
**Pull request by adroste**
Unfortunately, the current implementation ignores translations and the buttons texts are hardcoded inside the css. 
The easiest fix is to use the title attribute of the parent div that is already translated.

__Which issue does this PR address?__

Closes #43

### CODE REVIEW

**Comment by philippfromme** [comment]
Thanks for your contribution. I will have a look. 👍

File: assets/diagram-js-minimap.css [hunk 1]
```diff
@@ -31,14 +31,13 @@
...
-  height: 46px;
-  line-height: 46px;
+  padding: 10px;
   text-align: center;
 }
 
-.djs-minimap:not(.open) .toggle:before {
-  content: 'Open';
+.djs-minimap:not(.open) .toggle:before,
+.djs-minimap.open .toggle:before {
```
Comment (barmac) [inline assets/diagram-js-minimap.css:39]: How about if we simplify it a bit?
```suggestion
.djs-minimap .toggle:before {
```

**Comment by barmac** [comment]
I like the solution and left just one small suggestion about code readability. Thanks for you contribution! :)

File: assets/diagram-js-minimap.css (hunk 1 above)
Comment (adroste) [reply assets/diagram-js-minimap.css:39]: Actually, I did it exactly that way in the project I am working currently working on.
I just did not want to break other css rules or plugins by changing the specificity.
Maybe you should not simplify it, cause it may be a breaking change for some people...

Comment (barmac) [reply assets/diagram-js-minimap.css:39]: I understand your concern, but cannot really think of an example where lowering specificity could be a breaking change. I imagine that if somebody overrides provided styles, custom CSS should have higher or at least equal specificity. Then, if we lower our styles' specificity, custom CSS should remain to work.

File: assets/diagram-js-minimap.css [hunk 2]
```diff
@@ -31,14 +31,13 @@
 }
 
 .djs-minimap:not(.open) .toggle {
-  width: 46px;
-  height: 46px;
-  line-height: 46px;
+  padding: 10px;
```
Comment (barmac) [inline assets/diagram-js-minimap.css:34]: This is not related to the translation issue, right?

Comment (adroste) [reply assets/diagram-js-minimap.css:34]: Yes it is. Fixed width does not work, otherwise you end up with an overflowing box and/or weird spacing. I replaced the line-height/height with padding for simplicity. 
If you want to maintain the 46px height, just keep the line-height/height and only set padding to left/right.

File: assets/diagram-js-minimap.css (hunk 1 above)
Comment (adroste) [reply assets/diagram-js-minimap.css:39]: You are right.

File: assets/diagram-js-minimap.css (hunk 2 above)
Comment (barmac) [reply assets/diagram-js-minimap.css:34]: All right. So we should never have had fixed width here.

**Comment by barmac** [comment]
I will be happy to merge this today, if you could only squash the changes and adjust the commit message to our [contributing guidlines](https://github.com/bpmn-io/.github/blob/43e9cf462d38c36258f124e0692763c65e7607f0/.github/CONTRIBUTING.md#creating-a-pull-request). Thanks!

**Comment by barmac** [comment]
> I will be happy to merge this today, if you could only squash the changes and adjust the commit message to our [contributing guidlines](https://github.com/bpmn-io/.github/blob/43e9cf462d38c36258f124e0692763c65e7607f0/.github/CONTRIBUTING.md#creating-a-pull-request). Thanks!

Already merged. Have a look at https://github.com/bpmn-io/diagram-js-minimap/commit/65f32770710a5a52869a206b869ac4d162fe7de9 if you want to check what I meant :)

And thanks for your contribution!

**Comment by philippfromme** [comment]
Thx @barmac ❤️

**Comment by barmac** [comment]
Released in [v2.0.4](https://github.com/bpmn-io/diagram-js-minimap/commit/3d1a07e46cf67d4040eb90460b1be75b759a0ade) :)
